src/app/containers/ArticleTimestamp/index.test.jsx

```diff
@@ -34,13 +38,16 @@ describe('ArticleTimestamp', () => {
   });
 
   describe('daylight savings time', () => {
-    const daylightSavingsBehaviour = ({ descriptor, dateTime, longName }) => {
+    const daylightSavingsBehaviour = ({ descriptor, date, longName }) => {
       it(`should produce ${descriptor} as a descriptor when in ${longName}`, () => {
-        Date.now = jest.fn(() => dateTime);
-        const moreThanTenHoursAgo = timestampGenerator({
-          hours: 10,
-          seconds: 25,
+        const [
+          moreThanTenHoursAgo,
+          currentTimestamp,
```

Comment:
Instead of referring to this final return value as being `currentTimestamp` throughout this file, it would be more accurate to call this `mockCurrentTimestamp`, so it's clear that it isn't based on the actual timestamp when the test is run.  This value is the start of current day plus the first interval provided.